### DISCUSSION TITLE
QQ频道适配器收不到私信消息

### DISCUSSION THREAD

**Discussion by YangRucheng** [nonebot/discussions#336] [问答] [answered]
QQGuild适配器, 收不到私信消息 

(沙盒频道)

**Comment by yanyongyu** [comment, marked as answer]
私聊只有审核通过并在非沙盒中可以使用，请参考频道开发文档

https://bot.q.qq.com/wiki/develop/api/gateway/direct_message.html#%E7%A7%81%E4%BF%A1%E6%B6%88%E6%81%AF%E4%BA%8B%E4%BB%B6

**Reply by YangRucheng** [reply]
今天审核通过了, 切换到了正式环境, 仍然收不到私信, 是还有什么地方需要设置吗?
谢谢!

**Reply by yanyongyu** [reply]
intent设置了吗

**Reply by YangRucheng** [reply]
谢谢! 原来是这样
我刚才注意到config intent中还有字段可以设置
